### PR TITLE
[TSC] Create RC2 workspace

### LINKED ISSUE #955: [TSC] we need a proposal for RC-2 new WS
We need a proposal for the new WS RC-2.

This should follow the process defined in https://github.com/cntt-n/CNTT/blob/master/doc/gov/chapters/chapter03.md#34-new-work-streams-policy

Put the proposal to etherpad or any other online editing tool or write directly here.
https://etherpad.opnfv.org/p/RC2

### PR DESCRIPTION
**Pull request by xmulligan**
Closes #955

### CODE REVIEW

File: doc/ref_cert/RC2/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+[<< Back](../)
+
+# Reference Certification - Kubernetes based
```
Comment (rabi-abdel) [inline doc/ref_cert/RC2/README.md:3]: Can we change Certification -> Conformance?

File: doc/ref_cert/README.md [hunk 2]
```diff
@@ -6,3 +6,4 @@
 
 ## Available Programs
```
Comment (rabi-abdel) [inline doc/ref_cert/README.md:7]: @xmulligan can you also please edit RC2 chapter owenership in CODEOWNERS the same way other WS are defined in there?

File: CODEOWNERS [hunk 3]
```diff
@@ -89,3 +89,15 @@
 /doc/ref_impl/cntt-ri2/chapters/chapter07.md        @rabi-abdel @tomkivlin
 /doc/ref_impl/cntt-ri2/chapters/chapter08.md        @rabi-abdel @tomkivlin
 /doc/ref_impl/cntt-ri2/chapters/chapter09.md        @rabi-abdel @tomkivlin
+
+# Reference Conformance 2
+/doc/ref_cert/RC2/chapters/chapter01.md        @xmulligan @tomkivlin
```
Comment (rabi-abdel) [inline CODEOWNERS:94]: @xmulligan current process requires both WSL and TSL to approve a PR for merging. for consistency, would you mind having you and I (rabi-abdel) in the CODEOWNERS file.

Comment (xmulligan) [reply CODEOWNERS]: done

**Pull request merged**